tla/issue.md

```diff
@@ -0,0 +1,150 @@
+# dBFT 2.0 formal modelling and the liveness problem investigation
+
+### Summary or problem description
+
+This issue is aimed to emphasize the dBFT 2.0 liveness lock problem mentioned in
+https://github.com/neo-project/neo/issues/2029#issuecomment-725612388 and in the
+set of other related issues, PRs and papers with the help of TLA+ formal
+modelling tool. In this issue we'll present the formal algorithm specification
+and discuss the model checking results.
```

Comment:
... specification with a set of identified problems and propose ways to fix them in so-called dBFT 2.1.